packages/in_app_purchase/in_app_purchase_storekit/darwin/Classes/FIAObjectTranslator.h

```diff
@@ -56,6 +57,9 @@ NS_ASSUME_NONNULL_BEGIN
                                                   withError:(NSString *_Nullable *_Nullable)error
     API_AVAILABLE(ios(12.2));
 
++ (nullable SKPaymentTransactionMessage *) convertTransactionToPigeon:(SKPaymentTransaction *) transaction;
```

Comment:
These could have their own tests.  Not a huge deal if they have transitive coverage.